pitop/robotics/configurations.py

```diff
@@ -0,0 +1,23 @@
+from json import load
+from pathlib import Path
+
+from pitop import Pitop
+
+
+def alex_configuration():
+    return __load_json('alex.json')
+
+
+def AlexRobot():
```

Comment:
Should we delete this? In the Alex examples we're now using `Pitop.from_dict(alex_configuration())`

Reply:
Keep it for legacy support. We won't advertise it.